### PR TITLE
Created Endpoints for Mentor and Mentee Read

### PR DESCRIPTION
**Pull request by AdamHVu**
## Description

Created Endpoints for Mentors and Mentees to check their data, in this case using a "read" endpoint, given specific queries.
This was made in conjunction with Create and Update so that users could submit their information, check if it needs updates, and of course update it accordingly.

## Type of change
 [X ] New feature

## Checklist:

- [X ] My code follows PEP8 style guide
- [X ] I have removed unnecessary print statements from my code
- [X ] I have made corresponding changes to the documentation if necessary
- [X ] My changes generate no errors
- [X ] No commented-out code
- [X ] Size of pull request kept to a minimum
- [X ] Pull request description clearly describes changes made & motivations for said changes

My Loom Video: https://www.loom.com/share/4129501ebea2429e8495c0cb7764c56f

### CODE REVIEW

File: app/api.py [hunk 1]
```diff
@@ -84,25 +68,6 @@ async def read(data: Optional[Dict] = None):
     return {"result": API.db.read("Mentees", data)}
 
 
-
-
```
Comment (mikayla-kosmala) [inline app/api.py]: We do not want to remove other endpoints from this file.

**Review by mikayla-kosmala — CHANGES_REQUESTED** [review]
We do not want to remove other endpoints from this file. I see a lot of red and it is scaring me that if these changes get merged that these endpoints in red will be lost

File: app/api.py (hunk 1 above)
Comment (AdamHVu) [reply app/api.py]: Fixed, yeah UI on Github side threw me for a loop. Changes made.

**Review by GermanParra — APPROVED** [review]

**Review by mikayla-kosmala — APPROVED** [review]
Looks great! #adamskywalker

**Pull request merged**